webapp/scripts/mapbox.js

```diff
@@ -678,343 +678,883 @@
+        .setHTML('<h3>Cafeteria</h3><p>Our cafeteria provides a variety of meals and snacks prepared by Bon Appetit. There are vegetarian, vegan, and gluten-free options.</p>')
+        .addTo(map);
+        }
+        if(u == "Campus Center")
+        {
+        map.flyTo({center: [-122.284934,37.517124],speed: 0.3});
+        var popup = new mapboxgl.Popup({closeOnClick: true})
+        .setLngLat([-122.284934,37.517124])
+        .setHTML('<h3>Campus Center</h3><p>The Campus center is comprised of the Academic Success Center, Admissions Office, International Student Office, and Student Life and Leadership Office</p>')
         .addTo(map);
+        }
     }
 
     function pressBtn7(){
-        var u = document.getElementById('button_press7').value;
+        var u = document.getElementById('location6').innerHTML;
+        
+        if(u == "The Carl and Celia Berta Gellert Library")
+        {
         map.flyTo({center: [-122.284719, 37.51755], speed: 0.3});
         var popup = new mapboxgl.Popup({closeOnClick: true})
         .setLngLat([-122.284719, 37.51755])
-        .setHTML('<h3>Carl and Celia Berta Gellert Library</h3><p>The library offers a place to study and access to books, periodicals, journals, and online databases. Reference librarians are available to assist with research and other needs</p>')
+        .setHTML('<h3>The Carl and Celia Berta Gellert Library</h3><p>The library offers a place to study and access to books, periodicals, journals, and online databases. Reference librarians are available to assist with research and other needs</p>')
         .addTo(map);
+        }
+        if(u == "Career Services")
+        {
+        map.flyTo({center: [-122.284934,37.517124],speed: 0.3});
+        var popup = new mapboxgl.Popup({closeOnClick: true})
+        .setLngLat([-122.284934,37.517124])
+        .setHTML('<h3>Career Services</h3><p>Career Services provides career counseling, for-credit classes, information resources and tools, and on-campus events aimed at fostering the career development of students and alumni.</p>')
+        .addTo(map);
+        }
     }
 
     function pressBtn8(){
-        var u = document.getElementById('button_press8').value;
+        var u = document.getElementById('location7').innerHTML;
+        
+        if(u == "Library Lawn")
+        {
         map.flyTo({center: [-122.284331,37.517219], speed: 0.3});
         var popup = new mapboxgl.Popup({closeOnClick: true})
         .setLngLat([-122.284331,37.517219])
         .setHTML('<h3>Library Lawn</h3><p>School and Student events are held on the library lawn throughout the year. Here you have a view of the chapel, library, cafeteria, and Saint Joe’s residence hall.</p>')
         .addTo(map);
-    }
-
-    function pressBtn9(){
-        var u = document.getElementById('button_press9').value;
-        map.flyTo({center: [-122.284331,37.518310],speed: 0.3});
+        }
+        if(u == "Center for Spirituality")
+        {
```

Comment:
In JavaScript it's safer to open brackets right after a conditional or control statements because it can try to insert semicolons.  

ie.
```if (someCondition) {
    //Do Something
}```